src/scss/skin-modern/components/subtitlesettings/_subtitleoverlay-settings.scss

```diff
@@ -0,0 +1,134 @@
+@import '../../variables';
+@import '../../mixins';
+
+.#{$prefix}-ui-subtitle-overlay {
+
+  $colors: (
+    'white': #fff,
+    'black': #000,
+    'red': #f00,
+    'green': #0f0,
+    'blue': #00f,
+    'cyan': #0ff,
+    'yellow': #ff0,
+    'magenta': #f0f,
+  );
+
+  $opacities: (
+    100: 1,
+    75: .75,
+    50: .5,
+    25: .25,
+  );
+
+  $character-edges: (
+    'none': 'none',
+    'raised': '0px 0px 4px rgba(0, 0, 0, 0.9), 0px 1px 4px rgba(0, 0, 0, 0.9), 0px 2px 4px rgba(0, 0, 0, 0.9)',
+    'depressed': 'rgba(0, 0, 0, 0.8) 0px -2px 1px',
+    'uniform': '-2px 0px 1px rgba(0, 0, 0, 0.8), 2px 0px 1px rgba(0, 0, 0, 0.8), 0px -2px 1px rgba(0, 0, 0, 0.8), 0px 2px 1px rgba(0, 0, 0, 0.8), -1px 1px 1px rgba(0, 0, 0, 0.8), 1px 1px 1px rgba(0, 0, 0, 0.8), 1px -1px 1px rgba(0, 0, 0, 0.8), 1px 1px 1px rgba(0, 0, 0, 0.8)',
+    'dropshadowed': ' 0px 2px 1px rgba(0, 0, 0, 0.8)',
+  );
+
+  $font-sizes: (
+    50: .5,
+    75: .75,
+    100: 1,
```

Comment:
It still is. The default is set to `1.2em` on the `SubtitleOverlay`. These sizes are set on the `SubtitleLabel` child elements. To the final size is `1.2em * #{$font-size}em`.